app/controllers/application_controller/ci_processing.rb

```diff
@@ -548,7 +548,7 @@ def evacuatevms
     recs = find_checked_items
     recs = [params[:id].to_i] if recs.blank?
     @record = find_by_id_filtered(VmOrTemplate, recs.first)
-    if @record.is_available?(:evacuate) && !@record.ext_management_system.nil?
+    if @record.supports_evacuate? && !@record.ext_management_system.nil?
```

Comment:
yeah, but as you can see, the check is more advanced 
**host.acts_as_ems?**
and this is the place where this kind of logic should be and not in the UI